examples/echo_multiline.roc

```diff
@@ -0,0 +1,31 @@
+app [main!] { pf: platform "../platform/main.roc" }
+
+import pf.Stdin
+import pf.Stdout
+
+# Demonstrates: Reading multiline input from stdin until EOF, while loops, for loops, List.append
+
+main! : List(Str) => Try({}, [Exit(I32)])
+main! = |_args| {
+    var $lines = []
+    var $continue = True
+
+    # Read all lines from stdin until EOF (which returns empty string)
```

Comment:
Is it possible to differentiate between an actual empty line and EOF, to know when an input has been exhausted?